backend/codesirius/problems/views/tag.py

```
@@ -0,0 +1,82 @@
+            )
+        raise ValidationError(serializer.errors)
+
+
+class TagRetrieveUpdateDestroyAPIView(APIView):
+    permission_classes = [IsAuthenticatedOrReadOnly]
+
+    def get(self, request: Request, pk: int) -> CodesiriusAPIResponse:
+        try:
+            tag = Tag.objects.get(pk=pk)
+        except Tag.DoesNotExist:
+            raise NotFound("Tag not found")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)